### PR TITLE
[incubator/goldfish] Goldfish tweaks

### PR DESCRIPTION
**Pull request by mlaccetti**
**What this PR does / why we need it**:

Update the Goldfish chart to use the official Docker image, and tweaks accordingly.

**Which issue this PR fixes**:

Fix as PR.

**Special notes for your reviewer**:

Yay?

### CODE REVIEW

**Comment by mlaccetti** [comment]
@unguiculus seems that my rebase went sideways again, but I don't quite understand where/why; aside from that, should be good to go.

**Comment by unguiculus** [comment]
Doesn't look good. You did something wrong rebasing. Next time, please merge, which make s reviewing easier. PRs are squashed anyways.

**Comment by mlaccetti** [comment]
@unguiculus Fixed the horrorshow rebase!

File: incubator/goldfish/values.yaml [hunk 1]
```diff
@@ -3,16 +3,14 @@
 # Declare variables to be passed into your templates.
 replicaCount: 1
 image:
-  repository: quay.io/tuannvm/goldfish
-  tag: v0.7.3
-  pullPolicy: IfNotPresent
+  repository: caiyeon/goldfish
+  tag: 0.9.0
+  pullPolicy: Always
```
Comment (unguiculus) [inline incubator/goldfish/values.yaml]: Keep `IfNotPresent`.

**Comment by unguiculus** [comment]
cc @tuannvm

**Review by unguiculus — CHANGES_REQUESTED** [review]
You added yourself as a maintainer. Are you going to keep contributing to the chart and also to review PRs?

File: incubator/goldfish/values.yaml (hunk 1 above)
Comment (mlaccetti) [reply incubator/goldfish/values.yaml]: Yep, I'm planning to help out with that stuff.  Updated to use the `IfNotPresent` policy.

**Review by unguiculus — APPROVED** [review]

**Pull request merged**